ansible/roles/atmo-realvncserver/tasks/main.yml

```diff
@@ -98,15 +99,17 @@
         dest: /usr/share/backgrounds/default.jpg
         remote_src: True
       when: ansible_distribution == "Ubuntu" and ansible_distribution_major_version < "16"
+      ignore_errors: yes
 
     - name: Change desktop background on Ubuntu 16+
       copy:
         src: /usr/share/backgrounds/cloud_background.jpg
         dest: /usr/share/backgrounds/xfce/xfce-teal.jpg
         remote_src: True
       when: ansible_distribution == "Ubuntu" and ansible_distribution_major_version >= "16"
+      ignore_errors: yes
```

Comment:
In other parts of the codebase we use `failed_when`.. If you are curious (like I was) what the differences were between the two of them and when they were introduced into ansible: http://docs.ansible.com/ansible/playbooks_error_handling.html#controlling-what-defines-failure